exporter/management/commands/flattener.py

```diff
@@ -49,7 +49,7 @@ def callback(state, channel, method, properties, input_message):
                     shutil.copyfileobj(infile, outfile)
 
             excel = tmpfile.stat().st_size < settings.EXPORTER_MAX_JSON_BYTES_TO_EXCEL
-            output = flatterer.flatten(tmpfile.path, tmpdirname, xlsx=excel, json_stream=True, force=True)
+            output = flatterer.flatten(tmpfile.as_posix(), tmpdirname, xlsx=excel, json_stream=True, force=True)
```

Comment:
I guess `path` is not a method? Instead use `str(tmpfile)`. `as_posix()` does something a bit different on Windows, so it's a bit confusing to read.

Reply:
> I guess path is not a method?

Yes, it is not.
And done in https://github.com/open-contracting/data-registry/pull/235/commits/91bd4ba922c99a7c14a1c94d817ee7eeb265f88d